themes/scss/widgets/_chart.scss

```diff
@@ -6,7 +6,7 @@
   @include justify-content(space-between, justify);
   @include align-items(flex-end);
   width: 100%;
-  height: 75px; // match real widget content height, to avoid "jumpy" behavior
+  height: 107px; // match real widget content height, to avoid "jumpy" behavior
```

Comment:
how does this behave being responsive? any chance that the title goes in two lines?